app/README.md

```diff
@@ -122,6 +123,7 @@ npm run test-watch -- pages
 
 - [TypeScript](https://www.typescriptlang.org/) is used for type checking.
   - `npm run ts:check` - Type checks all files
+  - `npm run i18n-types` - Updates the i18n TypeScript declaration. You only need to run this if you've added a new English locale file (JSON files in `public/locales/en`). This runs automatically when you start the development server or build the application.
```

Comment:
Definitely a "gotcha" but I think since we *should* be adding new files relatively rarely it's worth it.